src/fides/service/messaging/messaging_service.py

```diff
@@ -131,19 +133,33 @@ def send_verification_code(
             MessagingActionType.SUBJECT_IDENTITY_VERIFICATION,
             self.config_proxy.execution.subject_identity_verification_required,
         ):
+            # Get service type, falling back to email messaging config if not configured
+            service_type = self.config_proxy.notifications.notification_service_type
+            if not service_type:
+                # Fall back to email messaging config if available
+                service_type = get_email_messaging_config_service_type(db=self.db)
+                if not service_type:
+                    raise MessageDispatchException(
+                        "No notification service type configured."
+                    )
+
             verification_code = _generate_id_verification_code()
             request.cache_identity_verification_code(verification_code)
 
-            dispatch_message(
-                self.db,
-                action_type=MessagingActionType.SUBJECT_IDENTITY_VERIFICATION,
-                to_identity=to_identity,
-                service_type=self.config_proxy.notifications.notification_service_type,
-                message_body_params=SubjectIdentityVerificationBodyParams(
-                    verification_code=verification_code,
-                    verification_code_ttl_seconds=self.config.redis.identity_verification_code_ttl_seconds,
-                ),
-                property_id=property_id,
+            dispatch_message_task.apply_async(
```

Comment:
My only higher-level-suggestion might be to take a `MessageDispatchTaskParams` `@dataclass` (Pydantic or plain) as a parameter to make it more type-safe/obvious what's being passed in rather than use a `kwargs` dictionary but that's more of an overall observation about invoking Celery jobs and passing arguments to them than anything specific to this PR. 😄